asset/js/sites-filter.js

```diff
@@ -0,0 +1,63 @@
+const pluck = property => element => element[property];
+
+var filter_values = new Array();
+
+function ApplyFilter( value ) {
+    const grid = document.getElementById("site-grid").getElementsByClassName("site-wrapper");
+    $(grid).each(function() {
+        subjects = $(this).data("subject").split(" ");
```

Comment:
Hmm... that's a good question. I don't have a specific intention to make the subjects array global.

Let me try experimenting a bit...

Reply:
Yeah, it looks like adding `let` before these variable names will prevent them from being accessible in the console. I don't think there's a security risk here, the information being passed around is all public, and nothing is getting sent back to the server - so there isn't a risk that I can detect. But, it is still a code smell to have these available like this.

I'll push a commit adding `let` (since these need to update each time the filter gets changed).

Reply:
Glad my JS knowledge isn't quite as dusty as I thought (at least, not in this particular instance). 🙂